src/components/routes/RouteListEdit.vue

```diff
@@ -2,8 +2,8 @@
 /**
  * RouteListEdit Component
  *
- * @description * Use this component to render routes in a list view
- * for editing.
+ * @description * Use this component to render routes which can be edited
+ * by the user. The number of routes to log is defined by global config.
```

Comment:
Specify which global var name that is, please.

Reply:
Added.